platform/mime/core/platform-mime-core-impl/src/main/java/ddf/mime/mapper/MimeTypeToTransformerMapperImpl.java

```diff
@@ -40,7 +40,9 @@
   private static final Logger LOGGER =
       LoggerFactory.getLogger(MimeTypeToTransformerMapperImpl.class);
 
-  public MimeTypeToTransformerMapperImpl() {}
+  public MimeTypeToTransformerMapperImpl() {
+    // Does nothing as of now, used in blueprint
```

Comment:
Not sure if blueprint requires these empty constructors or not, hence why left it, but if blueprints does not require these empty constructor, we can remove them.